production/Dockerfile.prereq

```diff
@@ -26,20 +26,23 @@ ARG URL
 ######################### Installation Manager #############################
 
 # Install Installation Manager
-RUN wget -q $URL/Install_Mgr_v1.6.2_Lnx_WASv8.5.5.zip -O /tmp/IM.zip \
-    && mkdir /tmp/im &&  unzip -qd /tmp/im /tmp/IM.zip \
-    && /tmp/im/installc -acceptLicense -accessRights admin \
+RUN wget -q "$URL/Install_Mgr_v1.6.2_Lnx_WASv8.5.5.zip" -O /tmp/IM.zip
+
+RUN mkdir /tmp/im &&  unzip -qd /tmp/im /tmp/IM.zip
```

Comment:
Please recombine these back in to a single RUN command. I know it made it harder for you to debug but it reduces the number of layers in the image.